.github/workflows/docker-push-mirrored.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   build-docker:
     name: cd/Push mirrored docker images
     if: github.repository_owner == 'mattermost'
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
Will this pull _all_ latest, or just LTS? If it's all latest, we should hard set this to 24.04.

Reply:
This specific job, and the e2e-test ones, are basically utility bash scripts for which it doesn't really matter on which Ubuntu version they run. E.g. they don't build or test the mattermost binary themselves

Reply:
But to answer your question, the [Github Docs](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) only mention Ubuntu LTS version as available as runners. Moreover, `ubuntu-latest` currently points to Ubuntu 22.04, because 24.04 runners are still in beta, so they seem to be quite conservative about it.